headers/csps/chains/arbitrum.ts

```diff
@@ -0,0 +1,15 @@
+import { loadEnv } from 'vite'
+
+import type { Csp } from '../../types'
+
+const mode = process.env.MODE ?? process.env.NODE_ENV ?? 'development'
+const env = loadEnv(mode, process.cwd(), '')
+
+export const csp: Csp = {
+  'connect-src': [
+    env.VITE_ARBITRUM_NODE_URL!,
```

Comment:
for all csp files, remove `loadEnv` logic and replace with `import.meta.env.VITE_VAR`.

(add `"headers/csps"` to tsconfig.json `include` list)

Reply:
I lied, this fixed the type error, but indeed, the env vars are not set up in import.meta.env at this stage.